assembly/package-res/SpoonDebug.sh

```diff
@@ -31,7 +31,7 @@ done
 echo "-"
 SPOONDIR=$(dirname "$(readlink -f "$0")")
 echo Launching Spoon: "$SPOONDIR/spoon.sh $SPOON_OPTIONS"
-if [ "$SPOON_REDIRECT" -ne "1" ] ; then
+if [ "$SPOON_REDIRECT" != "1" ] ; then
```

Comment:
Unfortunately, the script starts with: 
#!/bin/sh
This is a bash specific fix. I believe to accept this change, we need the script to begin with #!/bin/bash